src/main/res/drawable/ic_file_type.xml

```diff
@@ -0,0 +1,6 @@
+<vector android:height="23dp" android:viewportHeight="550.801"
+    android:viewportWidth="550.801" android:width="23dp" xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Looking at other drawable files, `viewportWidth` and `viewportHeight` is usually 24 or 36. What's the benefit of using a very high and unusual value like this?